website/blog/2021-08-25-webdriverio-meet-serenity-js.md

```diff
@@ -0,0 +1,52 @@
+---
+title: WebdriverIO, meet Serenity/JS
+author: Jan Molak
+authorURL: https://www.linkedin.com/in/janmolak/
+authorImageURL: https://avatars.githubusercontent.com/u/1089173?s=400&v=4
+---
+
+## WebdriverIO, meet Serenity/JS!
+
+Wouldn't it be great if you could use WebdriverIO to create full-stack test automation frameworks that interact with any interface of your system, generate in-depth reports, and help you write test code that's easy to understand, maintain, and reuse across projects and teams?
+
+Now you can, thanks to [Serenity/JS](https://serenity-js.org)!
+
+### About Serenity/JS
+
+[Serenity/JS](https://serenity-js.org) is an open-source acceptance testing and integration framework, designed to help you achieve all that and more 🚀
+
+Serenity/JS provides [adapters](https://serenity-js.org/modules/) that make it easy to integrate your tests with [Web apps](https://serenity-js.org/modules/webdriverio/), [REST APIs](https://serenity-js.org/modules/rest/), [Node.js servers](https://serenity-js.org/modules/local-server/), and pretty much anything a Node.js program can talk to.
+
+Thanks to Serenity/JS [Screenplay Pattern](https://serenity-js.org/handbook/design/screenplay-pattern.html), you and your team will also have a simple, consistent, and async-friendly [programming model](https://serenity-js.org/handbook/thinking-in-serenity-js/index.html) across all those interfaces.
+
+Apart from integration with your system under test, Serenity/JS can also integrate with popular test runners such as [Cucumber](https://serenity-js.org/modules/cucumber/), [Jasmine](https://serenity-js.org/modules/jasmine/), [Mocha](https://serenity-js.org/modules/mocha/), [Protractor](https://serenity-js.org/modules/protractor/) and now also [WebdriverIO](https://serenity-js.org/modules/webdriverio/)!
+
+Better yet, Serenity/JS provides a unique [reporting system](https://serenity-js.org/handbook/reporting/index.html) to help you generate consistent test execution and feature coverage reports and living documentation of your system across all its interfaces and across its various test suites.
```

Comment:
Some folks will wonder how it is compatible with WebdriverIO reporters. I remember you mentioned they are compatible. Can we mention this here?